sensors/barometer_LoRa/barometer_LoRa.ino

```diff
@@ -0,0 +1,127 @@
+#include <SPL06-007.h>
+#include <Wire.h>
+
+#include <SPI.h>
+#include <lmic.h>
+#include <hal/hal.h>
+
+// LoRaWAN settings
+static const u1_t PROGMEM APPEUI[8] = { /*Fill me in*/ };
+void os_getArtEui(u1_t* buf) {
+  memcpy_P(buf, APPEUI, 8);
+}
+
+static const u1_t PROGMEM DEVEUI[8] = { /*Fill me in*/ };
+void os_getDevEui(u1_t* buf) {
+  memcpy_P(buf, DEVEUI, 8);
+}
+
+static const u1_t PROGMEM APPKEY[16] = { /*Fill me in*/ };
+void os_getDevKey(u1_t* buf) {
+  memcpy_P(buf, APPKEY, 16);
+}
+
+unsigned char payload[5];
+static osjob_t sendjob;
+
+const unsigned TX_INTERVAL = 30;
+
+// Pin mapping for TTGO ESP32
+const lmic_pinmap lmic_pins = {
+  .nss = 18,
+  .rxtx = LMIC_UNUSED_PIN,
+  .rst = 14,
+  .dio = { 26, 33, 32 }  // Pins for the Heltec ESP32 Lora board/ TTGO Lora32 with 3D metal antenna
+};
+
+void setup() {
+  Wire.begin();
+  Serial.begin(9600);
+  Serial.println(F("Starting"));
+
+  // LMIC init
+  os_init();
+  // Reset the MAC state. Session and pending data transfers will be discarded.
+  LMIC_reset();
+
+  SPL_init();
+  // Start job (sending automatically starts OTAA too)
+  do_send(&sendjob);
+}
+
+void onEvent(ev_t ev) {
+  Serial.print(os_getTime());
+  Serial.print(": ");
+  switch (ev) {
+    case EV_JOINED:
+      //Serial.println(F("EV_JOINED"));
```

Comment:
We should remove all the commented out print statements

Reply:
Updated